src/modules/form/components/DynamicField.tsx

```diff
@@ -96,7 +97,15 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
       itemChanged({ linkId, value, type: ChangeType.User }),
     [linkId, itemChanged]
   );
-  const isDisabled = disabled || inputProps?.disabled;
+
+  const { user: currentUser } = useAuth();
+  const userCanEdit = !currentUser
+    ? false
+    : item.editorUserIds
+      ? item.editorUserIds.includes(currentUser.id)
+      : true;
```

Comment:
This is fine as is but I do struggle a little bit to read nested ternary operators. Maybe this is a little more readable. I think it's equivalent but I haven't tested it
```suggestion
  const userCanEdit = Boolean(currentUser?.id) && (!item.editorUserIds || item.editorUserIds.includes(currentUser.id));
```

Reply:
Thanks, yes this is equivalent (I tested locally) and much nicer. I used `!!currentUser?.id` instead of `Boolean(currentUser?.id)` since it seems to keep typescript happy.